packages/@react-types/table/src/index.d.ts

```diff
@@ -56,9 +56,11 @@ export interface SpectrumColumnProps<T> extends ColumnProps<T> {
   hideHeader?: boolean
 }
 
-export interface TableBodyProps<T> extends AsyncLoadable {
+export interface TableBodyProps<T> extends Omit<AsyncLoadable, 'isLoading'> {
```

Comment:
hmm ideally we'd change `AsyncLoadable` to have `loadingState` but I guess that would be a breaking change for other components.